dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/util/DateTimeUtils.java

```diff
@@ -27,7 +27,7 @@
 
 public final class DateTimeUtils {
 
-    private static final DateTimeFormatter DEFAULT_FORMAT = DateTimeFormatter.ISO_LOCAL_DATE_TIME;
+    private static final DateTimeFormatter DEFAULT_FORMAT = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss");
```

Comment:
Should we include the time zone?  Or is this what was being displayed before?

Reply:
Before it was hard coded, Java is super flexible so I we can make it whatever we want.

Reply:
I guess my question is, does the new format match the old hard coded value?